frontend/src/components/job/List.tsx

```diff
@@ -111,14 +120,13 @@ export function JobsListRenderer(props: JobsListRendererProps) {
           id: 'duration',
           label: t('translation|Duration'),
           getValue: job => {
-            const startTime = job.status?.startTime;
-            const completionTime = job.status?.completionTime;
-            if (!!startTime && !!completionTime) {
-              const duration = new Date(completionTime).getTime() - new Date(startTime).getTime();
+            const duration = getJobDuration(job);
```

Comment:
you could use a getDuration method in the job class.

```ts
const duration = job.getDuration();
```

Reply:
Thanks for pointing this out, didn't realize the logic was being duplicated in this file